src/screens/login/login.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { View, Image, Text } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+import { LoginForm } from 'src/screens/login/components';
+import { styles } from './styles';
+import { AppRoute, AuthRoute, ButtonMode } from 'src/common/enums';
+import { MainButton } from 'src/components';
+
+const LoginScreen: React.FC = () => {
+  const navigation = useNavigation();
+
+  const handleNavigateTo = (route: AuthRoute | AppRoute) => {
+    navigation.navigate(route);
+  };
```

Comment:
It has wrong type now
```suggestion
  const handleSignUpPress = () => {
    navigation.navigate({
      name: AuthRoute.SIGN_UP as never,
      params: {} as never,
    });
  };
```